### DISCUSSION TITLE
HTML

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6327] [Help]
Hello everyone . is it possible to insert html code  ?

**Comment by rafikApp** [comment]
I want to insert a 3d HTML page.
Like this one 



![Capture (1)](https://user-images.githubusercontent.com/106697208/173795037-88c343b2-ab9b-4439-b6a7-84adc0f12008.PNG)

**Reply by JoaoLCom** [reply]
great!

**Comment by melohagan** [comment]
Hi @rafikApp 

You could try using an <a href="https://docs.budibase.com/docs/embed">Embed</a> component.

**Reply by rafikApp** [reply]
Yes It works thank you. 
I have some buttons inside the html code, can I customize them to use the actions available in budibase ?

**Reply by melohagan** [reply]
This hack may be worth having a look at: https://github.com/Budibase/budibase/discussions/5112#discussioncomment-2456819

If you had a hidden button with an action, then you could click that button by getting the element by class & text value maybe.
